.circleci/config.yml

```diff
@@ -0,0 +1,54 @@
+version: 2
+jobs:
+
+  build:
+    docker:
+      - image: circleci/openjdk:8-node
+    steps:
+      - checkout
+
+      - restore_cache:
+          key: gradle-wrapper-{{ checksum "gradle/wrapper/gradle-wrapper.properties" }}
+      - restore_cache:
+          key: gradle-cache-{{ checksum "versions.props" }}-{{ checksum "build.gradle" }}
+
+      - run: ./gradlew --parallel resolveConfigurations
+
+      - save_cache:
+          key: gradle-wrapper-{{ checksum "gradle/wrapper/gradle-wrapper.properties" }}
+          paths:
+            - ~/.gradle/wrapper
+      - save_cache:
+          key: gradle-cache-{{ checksum "versions.props" }}-{{ checksum "build.gradle" }}
+          paths:
+            - ~/.gradle/caches
+
+      - run: ./gradlew publishToMavenLocal --parallel  # ensure artifacts are available to tests
+      - run: ./gradlew clean build
```

Comment:
`--parallel`? and why do we need the clean

Reply:
--parallel resulted in builds hanging at >8 minutes.  No idea why the clean was there.